File: Native Web Tech (C#)/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace NativeWebTech
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            webBrowser1.Navigate(Application.StartupPath + "/webview/index.html");
        }

        private void timer1_Tick(object sender, EventArgs e)
        {
            if (webBrowser1.DocumentTitle.Contains("close"))
            {
                this.Close();
            }
        }
    }
}

File: Native Web Tech (VB)/Form1.vb
Public Class Form1
    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        WebBrowser1.Navigate(Application.StartupPath.ToString & "/webview/index.html")
    End Sub

    Private Sub Timer1_Tick(sender As Object, e As EventArgs) Handles Timer1.Tick
        If WebBrowser1.DocumentTitle.Contains("close") Then
            Me.Close()
        End If
    End Sub
End Class
